test/tests/serial/user.bats

```diff
@@ -0,0 +1,42 @@
+load "${REPOSITORY_ROOT}/test/helper/common"
+load "${REPOSITORY_ROOT}/test/helper/setup"
+
+BATS_TEST_NAME_PREFIX='[User] '
+CONTAINER_NAME='dms-test_User_change_uid'
```

Comment:
Convention is `dms-test_` + `test-name-here`, where a `_` is used to delimit between these two (_and an optional third value for variants_), but the separate values are kebab case.

```suggestion
CONTAINER_NAME='dms-test_env-change-vmail-id'
```